### ISSUE TITLE
Compile Error: Symbol clash on Windows ( IGNORE )

### ISSUE THREAD

**Issue by bangusi** [vincentlaucsb/csv-parser#96]
version 1.3.1
on Windows 10

```
enum class VariableColumnPolicy {
        THROW = -1,
        IGNORE = 0,
        KEEP   = 1
    };
```

The constant IGNORE clashes with

``#define IGNORE              0       // Ignore signal``

In  WinBase.h

**Comment by vincentlaucsb** [comment]
Thanks for reporting it. I've confirmed it on my end.

**Comment by vincentlaucsb** [comment]
Fixed by 1.3.3
